include/revng-c/mlir/Pipes/MLIRContainer.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+//
+// Copyright (c) rev.ng Labs Srl. See LICENSE.md for details.
+//
+
+#include "llvm/ADT/DenseMap.h"
+
+#include "mlir/IR/BuiltinOps.h"
+#include "mlir/IR/FunctionInterfaces.h"
+#include "mlir/IR/MLIRContext.h"
+
+#include "revng/Pipeline/Container.h"
+
+namespace revng::pipes {
+
+class MLIRContainer : public pipeline::Container<MLIRContainer> {
+public:
+  static const char ID;
+  static constexpr auto Name = "mlir-module";
+  static constexpr auto MIMEType = "text/mlir";
```

Comment:
1. Currently text because it's easier to debug.
2. I didn't find anything official. `text/mlir` was already listed here: https://docs.rev.ng/references/mime-types/

Reply:
1. I understand, but these choices tend to stick around forever. Let's switch to byte code. I think you can get the textual representation just by doing `mlir-opt -S` or something like that.
2. Unless we find some official source, I'd go with `application/x.mlir.bc` for MLIR byte code and with `text/x.mlir.ir`, just for coherence with what we're using for LLVM IR.

Reply:
Done.